docs/migration/migrate_6_0.asciidoc

```diff
@@ -4,13 +4,18 @@
 This section discusses the changes that you need to be aware of when migrating
 your application to Kibana 6.0.
 
+[float]
+=== Types removed from Kibana index
+*Details:* With the imminent {es-ref}/removal-of-types.html[removal of types in Elasticsearch], we have updated our index to not rely on types. Kibana 5.6.x can support both of these mappings to minimize downtime. A UI is provided with X-Pack to assist with the migration of the Elastic stack.
+
+*Impact:* You will need migrate your Kibana index either {kibana-ref}/migrating-6.0-index.html[manually], or through the Migration UI in X-Pack.
```

Comment:
I believe @archanid is still working on that. We should link when it becomes available.